ntbs-service/Models/Entities/DataQualityDotVotAlert.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq.Expressions;
+using ntbs_service.Helpers;
+using ntbs_service.Models.Enums;
+
+namespace ntbs_service.Models.Entities
+{
+    public class DataQualityDotVotAlert : Alert
+    {
+        public static readonly Expression<Func<Notification, bool>> NotificationQualifiesExpression =
+            n =>
+                // record has one or more social risk factors but DOT is set to No
+                ((
+                    n.SocialRiskFactors.AlcoholMisuseStatus == Status.Yes ||
+                    n.SocialRiskFactors.RiskFactorDrugs.Status == Status.Yes ||
+                    n.SocialRiskFactors.RiskFactorHomelessness.Status == Status.Yes ||
+                    n.SocialRiskFactors.RiskFactorImprisonment.Status == Status.Yes ||
+                    n.SocialRiskFactors.MentalHealthStatus == Status.Yes ||
+                    n.SocialRiskFactors.SmokingStatus == Status.Yes ||
+                    n.SocialRiskFactors.AsylumSeekerStatus == Status.Yes ||
+                    n.SocialRiskFactors.ImmigrationDetaineeStatus == Status.Yes
+                ) && n.ClinicalDetails.IsDotOffered == false)
+                || // OR - record has DOT set to Yes but no social risk factors
+                ((
+                    n.SocialRiskFactors.AlcoholMisuseStatus != Status.Yes ||
```

Comment:
should this be && based on comment above? -> 'but no social risk factors'
if it is then should this actually be just the opposite of the section above and we can move it to a variable `atLeastOneSocialRiskFactor` and `!atLeastOneSocialRiskFactor`

Reply:
gah! absolutely correct spot - I've actually got that fixed but my branch push failed 🤦‍♂ 
I just corrected that, sorry to make you review twice!